lib/runtime/procs/sort.js

```diff
@@ -58,8 +58,8 @@ var sort = fanin.extend({
         var k, row;
         var space_left = this.limit - this.n_pts;
         if (space_left < points.length) {
```

Comment:
I see there are similar issues w/ `space_left` and `limit` in the `sort` proc - I'll take care of them.